src/Compilers/CSharp/Portable/Declarations/DeclarationTreeBuilder.cs

```diff
@@ -810,7 +862,6 @@ private static ImmutableSegmentedHashSet<string> GetEnumMemberNames(SeparatedSyn
             bool anyMemberHasAttributes = false;
             foreach (var member in members)
             {
-                memberNamesBuilder.Add(member.Identifier.ValueText);
                 if (!anyMemberHasAttributes && member.AttributeLists.Any())
                 {
                     anyMemberHasAttributes = true;
```

Comment:
>anyMemberHasAttributes = true; [](http://example.com/codeflow?start=20&length=30)

Is there a good reason to continue the loop at this point?

Reply:
Nope!